tests/e2e/utils/lambda_layer/powertools_layer.py

```diff
@@ -16,9 +16,16 @@ class LocalLambdaPowertoolsLayer(BaseLocalLambdaLayer):
 
     def __init__(self, output_dir: Path = CDK_OUT_PATH):
         super().__init__(output_dir)
-        self.package = f"{SOURCE_CODE_ROOT_PATH}[pydantic]"
+        self.package = f"{SOURCE_CODE_ROOT_PATH}[all]"
         self.build_args = "--platform manylinux1_x86_64 --only-binary=:all: --upgrade"
```

Comment:
We're gonna need to extract the platform argument into a separate variable, so we can determine the correct value:

* **X86**: `manylinux1_x86_64`
* **ARM** `manylinux2014_aarch64`

It took me a while to figure out the oldest OS wheel distro supporting ARM that would work for X-Ray SDK dependency (`wrapt`).

My repro:

**ARM**

```bash
pip install .[all] --platform manylinux2014_aarch64 --only-binary=:all: -t /tmp/ptARM
ls -l /tmp/ptARM/wrapt/
```

**X86**

```bash
pip install .[all] --platform manylinux1_x86_64 --only-binary=:all: -t /tmp/ptX86
ls -l /tmp/ptX86/wrapt/
```